lib/geodb.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var geodb = require('fxa-geodb')()
```

Comment:
Yes, @jrgm was going to download the database separately, we need to [add a configuration option with the database path](https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js) and pass it to the fxa-geodb.